rancher/installing-rancher/installing-server/basic-ssl-config/index.md

```diff
@@ -48,11 +45,10 @@ server {
     ssl_certificate <cert_file>;
     ssl_certificate_key <key_file>;
 
-    proxy_buffering off;
-    proxy_buffer_size 512;
-
     location / {
-        proxy_set_header X-API-request-url $scheme://<host>$request_uri;
+        proxy_set_header Host $host;
```

Comment:
I think you want to also use:
`proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;`
So Cattle can look at the client ip.